ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less

```diff
@@ -78,17 +78,17 @@
   .dropdown-text {
     padding: 0px 10px;
     width: ~'calc(100% - @{minimumButtonSize})';
-    min-height: @minimumButtonSize;
+    height: @minimumButtonSize;
```

Comment:
Selected sources are separated by ` | ` and if needed, the list of sources would continue to the next line. I think this change limits the dropdown to a single line thus hiding part of the list.

Reply:
<img width="541" alt="screen shot 2018-05-02 at 8 59 03 am" src="https://user-images.githubusercontent.com/16843783/39534789-454fb686-4de7-11e8-9dff-6839f3446e8d.png">
This is showing with the min-height property set. Using height sizes it properly. I just checked and you were right about how the height property hides part of the selected list of sources.